### PR TITLE
First Contribution clarity edits

### PR DESCRIPTION
**Pull request by Crissman**
Edits of First Contribution tutorial for clarity.

### CODE REVIEW

File: docs/source/tutorial/first.rst [hunk 1]
```diff
@@ -22,11 +22,11 @@ Here, we use a very simple quadratic function as an example of objective functio
        ...:     return (x - 2) ** 2
        ...:
 
-Our goal is to find out ``x`` that minimizes the output of ``objective`` function, which we refer to as "optimization." During the optimization, Optuna repeatedly invokes and evaluates the objective function with different values of ``x``.
+This function returns the value of (x - 2) squared. Our goal is to find the value of ``x`` that minimizes the output of the ``objective`` function. This is the "optimization."  During the optimization, Optuna repeatedly calls and evaluates the objective function with different values of ``x``.
```
Comment (hvy) [inline docs/source/tutorial/first.rst]:
```suggestion
This function returns the value of :math:`(x - 2)^2`. Our goal is to find the value of ``x`` that minimizes the output of the ``objective`` function. This is the "optimization."  During the optimization, Optuna repeatedly calls and evaluates the objective function with different values of ``x``.
```

File: docs/source/tutorial/first.rst [hunk 2]
```diff
@@ -7,13 +7,13 @@ First Optimization
 Quadratic Function Example
 --------------------------
 
-Let us try very simple optimization in IPython shell.
+Usually, Optuna is used as a hyper-parameter optimizer, but as an example, let us try a direct optimization in IPython shell.
```
Comment (hvy) [inline docs/source/tutorial/first.rst]: Is `hyper-parameter optimizer` a common phrase?. Maybe it is just me but how about something along `Usually, Optuna is used to optimize hyper-parameters, ...`.

Comment (hvy) [inline docs/source/tutorial/first.rst]: Not sure the reader would clearly understand how `direct optimization` would differ from `hyper-parameter optimization`. How about `.. let us directly optimize a quadratic function to demonstrate its concept`, and delete the sentence below. The latter part also motivates why.

Comment (Crissman) [reply docs/source/tutorial/first.rst]: Agreed, and updated.

Comment (Crissman) [reply docs/source/tutorial/first.rst]: Updated.

File: docs/source/tutorial/first.rst [hunk 3]
```diff
@@ -7,13 +7,13 @@ First Optimization
 Quadratic Function Example
 --------------------------
 
-Let us try very simple optimization in IPython shell.
+Usually, Optuna is used to optimize hyper-parameter, but as an example, let us directly optimize a quadratic function in an IPython shell.
```
Comment (hvy) [inline docs/source/tutorial/first.rst]: A nit.
```suggestion
Usually, Optuna is used to optimize hyper-parameters, but as an example, let us directly optimize a quadratic function in an IPython shell.
```

File: docs/source/tutorial/first.rst [hunk 4]
```diff
@@ -22,11 +22,11 @@ Here, we use a very simple quadratic function as an example of objective functio
        ...:     return (x - 2) ** 2
        ...:
 
-Our goal is to find out ``x`` that minimizes the output of ``objective`` function, which we refer to as "optimization." During the optimization, Optuna repeatedly invokes and evaluates the objective function with different values of ``x``.
+This function returns the value of :math:`(x - 2)^2`. Our goal is to find the value of ``x`` that minimizes the output of the ``objective`` function. This is the "optimization."  During the optimization, Optuna repeatedly calls and evaluates the objective function with different values of ``x``.
```
Comment (hvy) [inline docs/source/tutorial/first.rst]:
```suggestion
This function returns the value of :math:`(x - 2)^2`. Our goal is to find the value of ``x`` that minimizes the output of the ``objective`` function. This is the "optimization". During the optimization, Optuna repeatedly calls and evaluates the objective function with different values of ``x``.
```

**Pull request merged**